service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43aSpecialGuardianshipOrderDocumentParameterGenerator.java

```diff
@@ -81,7 +82,20 @@ private String getSpecialGuardianAppointeeMessage(CaseData caseData, int numOfCh
         String childOrChildren = (numOfChildren == 1 ? "child" : "children");
         String applicant = appointedGuardianFormatter.getGuardiansNamesForDocument(caseData);
 
-        return format("The Court orders that %s appointed as special guardian for the %s.", applicant, childOrChildren);
+        ManageOrdersEventData eventData = caseData.getManageOrdersEventData();
+
+        StringBuilder sb = new StringBuilder();
+        if (isNotEmpty(applicant)) {
+            sb.append(format("The Court orders that %s appointed as special guardian for the %s.", applicant,
+                    childOrChildren));
+        }
+        if (isNotEmpty(eventData.getAppointedGuardianDetails())) {
```

Comment:
why do we need the getAppointedGuardianDetails() here too (trying to see from the ticket)? just wondering what it contains?